src/utils/__fixtures__/gcsFileObj.ts

```diff
@@ -0,0 +1,1950 @@
+// tslint:disable: object-literal-sort-keys
+// tslint:disable: max-file-line-count
+
+export const squirrelStatsGCSFileObj = {
```

Comment:
Good job with setting up all these fixtures 👍 
Do you think we need this object as a whole, though?
Can we perhaps kill (or add a stub) packageJson attributes at least? They are useless for our tests and make the fixture less readable.

Reply:
Sure. Can do. I originally had pulled out everything we didn't use, but then had second thoughts - maybe making it too clean would put us in danger of things passing which wouldn't otherwise, with a more realistic fixture? So then I just left everything in. (You're right, though, that the `packageJson` entries shouldn't be what makes the difference.)

Any tips on where the draw the line on questions like ^^^^ ?

Reply:
🤷‍♂ 
No fixture is ideal, but stripping down highly unrelated attributes while preserving the overall structure seems ok to me.

Reply:
Okay, got rid of duplicated stuff (of which there was a LOT) and `packageJson`. Cut the filesize by a factor of more than 10! 😮